addons/point_of_sale/models/pos_session.py

```diff
@@ -690,7 +690,9 @@ def _create_invoice_receivable_lines(self, data):
             account_id = commercial_partner.property_account_receivable_id.id
             receivable_lines = MoveLine.create(vals)
             for receivable_line in receivable_lines:
-                if (not receivable_line.reconciled):
+                if (not (receivable_line.company_currency_id.is_zero(receivable_line.debit)
+                         and receivable_line.company_currency_id.is_zero(receivable_line.credit)
+                )):
```

Comment:
how about checking the balance if it's zero?

Reply:
Apparently, it's not possible to have here credit = debit != 0. So, let's just check balance